src/Illuminate/Console/Concerns/ConfiguresPrompts.php

```diff
@@ -24,7 +24,9 @@ protected function configurePrompts(InputInterface $input)
     {
         Prompt::setOutput($this->output);
 
-        Prompt::fallbackWhen(! $input->isInteractive() || windows_os() || $this->laravel->runningUnitTests());
+        Prompt::interactive(($input->isInteractive() && stream_isatty(STDIN)) || $this->laravel->runningUnitTests());
+
+        Prompt::fallbackWhen(windows_os() || $this->laravel->runningUnitTests());
```

Comment:
Do we need to check for unit tests here? The same check in the previous if statement.

Reply:
Yes, otherwise, the Symfony fallback won't kick in and Prompts will try to render during test execution. We could solve that by making Prompts non-interactive when testing, but console test assertions like `expectsQuestion` wouldn't work.

The only solution currently is to force the Symfony fallback when running tests. The Prompts library would need its own high-level test assertions to avoid this. The current test helpers in Prompts are quite low-level and involve mocking key presses and asserting against rendered output, which is only intended for testing the Prompts library itself.

Reply:
I mean we can remove this check from this if statement, it will never be executed here because it exists in the previous statement.

Reply:
We need to enable interactive mode _and_ the fallback when running unit tests.

If we remove this check, it will only enable interactive mode, but not the fallbacks, when running unit tests.

Reply:
I see 👍